pkg/ruler/storage/wal/wal.go

```diff
@@ -66,7 +66,7 @@ type Storage struct {
 
 // NewStorage makes a new Storage.
 func NewStorage(logger log.Logger, metrics *Metrics, registerer prometheus.Registerer, path string) (*Storage, error) {
-	w, err := wlog.NewSize(logger, registerer, SubDirectory(path), wlog.DefaultSegmentSize, true)
+	w, err := wlog.NewSize(logger, registerer, SubDirectory(path), wlog.DefaultSegmentSize, wlog.CompressionSnappy)
```

Comment:
The boolean argument is now replaced with CompressionSnappy, CompressionZstd and CompressionNone. Please confirm if the choice to use CompressionSnappy here is correct